docs/04_cv32a65x_design/source/port_ras.rst

```diff
@@ -15,7 +15,7 @@
    * - Signal
      - IO
      - Description
-     - Connection
+     - connexion
```

Comment:
"Connexion" in French, "connection" in English. This is counter-intuitive.

Reply:
Oeufs corses !   Fixed